easybuild/easyblocks/i/icc.py

```diff
@@ -74,29 +75,35 @@ def install_step(self):
                 'license_file_name': LICENSE_FILE_NAME_2012,
             }
 
-        super(EB_icc, self).install_step(silent_cfg_names_map=silent_cfg_names_map)
+        cfg_extras_map = {}
+        if LooseVersion(self.version) >= LooseVersion('2016'):
+            cfg_extras_map = {
+                'COMPONENTS': 'ALL',
```

Comment:
I don't think this is a very good idea... This is icc, not "Intel world".

Is there documentation on what are valid values for `COMPONENTS`?